modules/cco-ccoctl-install-verifying.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * installing/validating-an-installation.adoc
+
+:_content-type: PROCEDURE
+[id="cco-ccoctl-install-verifying_{context}"]
+= Clusters that use short-term credentials: Verifying the credentials configuration
+
+You can verify that your cluster is using short-term, limited-privilege security credentials for individual components.
```

Comment:
noticed this should only say "short-term" since not all platforms are limited-priv yet